Article/Article.js

```diff
@@ -3,18 +3,42 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.domElement = domElement;
+    // create a reference to the ".expandButton" class.
+    this.expandButton = this.domElement.querySelector(".expandButton");
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = "Read more.";
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener("click", this.expandArticle.bind(this));
```

Comment:
This is a great use of bind.
 If you like you can as well use arrow functions to "skip" this passage and do it automatically in this fashion : `  this.expandButton.addEventListener("click", () => this.expandArticle());`